### PR TITLE
Add Choconum

### PR DESCRIPTION
**Pull request by nooges**
## Description

Adds Choconum keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/choconum/chconf.h [hunk 1]
```diff
@@ -0,0 +1,714 @@
...
+
+/**
+ * @brief   Core Memory Manager APIs.
+ * @details If enabled then the core memory manager APIs are included
+ *          in the kernel.
+ *
+ * @note    The default is @p TRUE.
+ */
+#if !defined(CH_CFG_USE_MEMCORE)
+#define CH_CFG_USE_MEMCORE                  FALSE
```
Comment (zvecr) [inline keyboards/keebio/choconum/chconf.h:334]: Per #8695
```suggestion
#define CH_CFG_USE_MEMCORE                  TRUE
```

File: keyboards/keebio/choconum/choconum.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+) \
+{ \
+    { KA1, KA2, KA3, KA4 }, \
+    { KB1, KB2, KB3, KB4 }, \
+    { KC1, KC2, KC3, KC4 }, \
+    { KD1, KD2, KD3, KD4 }, \
+    { KE1, KE2, KE3, KE4 } \
+}
+
+#define LAYOUT_numpad( \
```
Comment (zvecr) [inline keyboards/keebio/choconum/choconum.h:36]:
```suggestion
#define LAYOUT_numpad_5x4( \
```

File: keyboards/keebio/choconum/choconum.h [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_5x4( \
```
Comment (zvecr) [inline keyboards/keebio/choconum/choconum.h:21]:
```suggestion
#define LAYOUT_ortho_5x4( \
```

File: keyboards/keebio/choconum/info.json [hunk 4]
```diff
@@ -0,0 +1,64 @@
+{
+    "keyboard_name": "Choconum",
+    "url": "https://keeb.io",
+    "maintainer": "Keebio",
+    "width": 5,
+    "height": 4,
+    "layouts": {
+        "LAYOUT_numpad": {
```
Comment (zvecr) [inline keyboards/keebio/choconum/info.json:8]:
```suggestion
        "LAYOUT_numpad_5x6": {
```

File: keyboards/keebio/choconum/info.json [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+                {"x":0, "y":3},
+                {"x":1, "y":3},
+                {"x":2, "y":3},
+                {"x":3, "y":3, "h":2},
+
+                {"x":0, "y":4,"w":2},
+                {"x":2, "y":4}
+            ]
+        },
+        "LAYOUT_5x4": {
```
Comment (zvecr) [inline keyboards/keebio/choconum/info.json:34]:
```suggestion
        "LAYOUT_ortho_5x4": {
```

File: keyboards/keebio/choconum/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+    _BASE,
+    _FN
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [_BASE] = LAYOUT_numpad(
```
Comment (zvecr) [inline keyboards/keebio/choconum/keymaps/default/keymap.c:26]:
```suggestion
    [_BASE] = LAYOUT_numpad_5x4(
```

File: keyboards/keebio/choconum/keymaps/default/keymap.c [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_P0,            KC_PDOT
+    ),
+    [_FN] = LAYOUT_numpad(
```
Comment (zvecr) [inline keyboards/keebio/choconum/keymaps/default/keymap.c:33]:
```suggestion
    [_FN] = LAYOUT_numpad_5x4(
```

File: keyboards/keebio/choconum/keymaps/via/keymap.c [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /* Base */
+    [0] = LAYOUT_5x4(
```
Comment (zvecr) [inline keyboards/keebio/choconum/keymaps/via/keymap.c:20]:
```suggestion
    [0] = LAYOUT_ortho_5x4(
```

File: keyboards/keebio/choconum/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+        KC_NLCK, KC_PSLS, KC_PAST, KC_PMNS,
+        KC_P7,   KC_P8,   KC_P9,   KC_PPLS,
+        KC_P4,   KC_P5,   KC_P6,   KC_PEQL,
+        KC_P1,   KC_P2,   KC_P3,   KC_PENT,
+        KC_P0,   KC_P0,   KC_PDOT, KC_PENT
+    ),
+    [1] = LAYOUT_5x4(
```
Comment (zvecr) [inline keyboards/keebio/choconum/keymaps/via/keymap.c:27]:
```suggestion
    [1] = LAYOUT_ortho_5x4(
```

File: keyboards/keebio/choconum/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______
+    ),
+    [2] = LAYOUT_5x4(
```
Comment (zvecr) [inline keyboards/keebio/choconum/keymaps/via/keymap.c:34]:
```suggestion
    [2] = LAYOUT_ortho_5x4(
```

File: keyboards/keebio/choconum/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______,
+        _______, _______, _______, _______
+    ),
+    [3] = LAYOUT_5x4(
```
Comment (zvecr) [inline keyboards/keebio/choconum/keymaps/via/keymap.c:41]:
```suggestion
    [3] = LAYOUT_ortho_5x4(
```

File: keyboards/keebio/choconum/rules.mk [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+MIDI_ENABLE = no            # MIDI support
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no           # Audio output on port C6
+FAUXCLICKY_ENABLE = no      # Use buzzer to emulate clicky switches
+HD44780_ENABLE = no         # Enable support for HD44780 based LCDs
+
```
Comment (zvecr) [inline keyboards/keebio/choconum/rules.mk:23]:
```suggestion

LAYOUTS = ortho_5x4 numpad_5x4

```

File: keyboards/keebio/choconum/keymaps/default/config.h [hunk 13]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 Keebio
```
Comment (fauxpark) [inline keyboards/keebio/choconum/keymaps/default/config.h:1]: This file can be deleted.

File: keyboards/keebio/choconum/keymaps/via/config.h [hunk 14]
```diff
@@ -0,0 +1,19 @@
+/* Copyright 2020 Keebio
```
Comment (fauxpark) [inline keyboards/keebio/choconum/keymaps/via/config.h:1]: As above.

File: keyboards/keebio/choconum/readme.md [hunk 15]
```diff
@@ -0,0 +1,18 @@
+# Choconum
+
+![Choconum](https://cdn.shopify.com/s/files/1/1851/5125/products/image_ac483e26-c47b-4939-904f-41ad8de28210_1024x1024@2x.jpg?v=1586055933)
+
+A numpad for Kailh Choc switches
+
+* Keyboard Maintainer: [Keebio](https://github.com/yourusername)
```
Comment (Erovia) [inline keyboards/keebio/choconum/readme.md:7]: hmmmmmm....

Comment (nooges) [reply keyboards/keebio/choconum/readme.md:7]: 🤦‍♂

**Pull request merged**